--- .settings/org.eclipse.wst.common.component	2017-09-27 09:10:37.733478577 -0700
+++ .settings/org.eclipse.wst.common.component_modified	2017-09-26 10:07:19.661023723 -0700
@@ -6,6 +6,7 @@
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/java"/>
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/resources"/>
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/target/generated_jsp_classes"/>
         <property name="context-root" value="oscar"/>
         <property name="java-output-path" value="/oscar/target/classes"/>
     </wb-module>
